tests/framework/utils/snapshot.go

```diff
@@ -58,8 +62,15 @@ func (k8sh *K8sHelper) CheckSnapshotISReadyToUse(name, namespace string, retries
 
 // snapshotController creates or deletes the snapshotcontroller and required RBAC
 func (k8sh *K8sHelper) snapshotController(action string) error {
-	controller := fmt.Sprintf("%s/%s/%s", repoURL, snapshotterVersion, controllerPath)
-	_, err := k8sh.Kubectl(action, "-f", controller)
+	controllerURL := fmt.Sprintf("%s/%s/%s", repoURL, snapshotterVersion, controllerPath)
+	controllerManifest, err := k8sh.getManifestFromURL(controllerURL)
```

Comment:
For my own understanding: the `canary` version spec is coming from an external repo (`kubernetes-csi`) yes, not from the rook repo?

Reply:
Yes for some reason the released external-snapshotter uses a canary image tag, not the released version https://github.com/kubernetes-csi/external-snapshotter/blob/v3.0.0/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml#L28, the canary is built from the master branch, we should replace the canary with the released version tag.

Reply:
This seems like a bug in the snapshotter manifest, but we at least need to workaround for now...